pyproject.toml

```
@@ -136,7 +138,7 @@ good-names = [
 
 [tool.pytest.ini_options]
 minversion = "7.0"
-addopts = "-rs --cov=tripper --cov-report=term --doctest-modules"
+addopts = "-rs --cov=tripper --cov-report=term --doctest-modules --ignore=examples --ignore=tripper"
```

```suggestion
addopts = "-rs --cov=tripper --cov-report=term --doctest-modules --ignore=examples"
```
I also do not like ignoring of examples, but tripper which is our code-base should definitely not be ignored.

---

Replaced it with new `--doctest-ignore-import-errors`

---

> Replaced it with new `--doctest-ignore-import-errors`

---

> I also do not like ignoring of examples, but tripper which is our code-base should definitely not be ignored.

Replaced it with new `--doctest-ignore-import-errors`